app/actors/hyrax/actors/file_set_actor.rb

```diff
@@ -10,6 +10,40 @@ def initialize(file_set, user)
         @user = user
       end
 
+      # @!group Asynchronous Operations
+
+      # Spawns asynchronous IngestJob
+      # Called from FileSetsController, AttachFilesToWorkJob, ImportURLJob, IngestLocalFileJob
+      # @param [Hyrax::UploadedFile, File, ActionDigest::HTTP::UploadedFile, Tempfile] file the file uploaded by the user
```

Comment:
You can't accept Tempfile here. Because you can't run an asynchronous job on a file that is temporary.